docs/book/src/developer/tilt.md

```diff
@@ -257,7 +280,28 @@ create a cluster. There are [example worker cluster
 configs](https://github.com/kubernetes-sigs/cluster-api/tree/main/test/infrastructure/docker/examples) available.
 These can be customized for your specific needs.
 
-<aside class="note">
+### Deploying a workload cluster
+
+After your kind management cluster is up and running with Tilt, you can deploy a workload clusters in the Tilt web UI based off of YAML templates from specified directories. By default, templates are read from `./test/infrastructure/docker/templates`.
+
+These deployment resources are found in the Tilt web UI under the label grouping `<provider>-flavors` and `<provider>-clusterclasses` for each specified provider, i.e. `docker-flavors` and `docker-clusterclasses`. Note that deploying a workload cluster from Tilt UI is also termed as flavor cluster deployment.
+
+The `<provider>-flavors` category contains flavor templates, you can create a cluster by clicking "Create flavor cluster" or the clockwise arrow icon ⟳. Note that each time a flavor is deployed, it deploys a new workload cluster in addition to the existing ones. To delete clusteres based off of a flavor, click on "Delete \<flavor\> cluster," and click on "Delete all flavor clusters" to delete all workload clusters.
```

Comment:
```suggestion
The `<provider>-flavors` category contains cluster templates, you can create a cluster by clicking "Create cluster" or the clockwise arrow icon ⟳. Note that each time a cluster is deployed, it deploys a new workload cluster in addition to the existing ones. To delete clusters based off of a flavor, click on "Delete \<flavor\> cluster," and click on "Delete all flavor clusters" to delete all workload clusters.
```
I think we usually call those cluster templates.

Reply:
I would also suggest to call the tab in the Tilt UI `<provider>-clustertemplates`. I think this way we can avoid an awkward mix between flavor and cluster template, or call cluster templates flavors.